src/main/java/com/cburch/logisim/vhdl/sim/VhdlSimulatorTclBinder.java

```diff
@@ -139,6 +142,9 @@ public void start() {
 
                 vsim.tclStartCallback();
                 return;
+              } else if (line.contains("\"vcom -reportprogress 300 -work work ../src/LogisimVhdlSimComp")) {
```

Comment:
I do not like to much to check on a fixed String like this. I understand the idea, but is there not a better way to detect the end? Or if this is the only way, please add a final constant and make a remark in the code that this should always the last line.

Reply:
Basically (in my case) the error ends with : 

> \#     while executing
> \# "vcom -reportprogress 300 -work work ../src/LogisimVhdlSimComp_0.vhdl"

where the last line is the one generated at com/cburch/logisim/vhdl/sim/VhdlSimulatorTclComp.java:69 :

```
compFiles.append("vcom -reportprogress 300 -work work ../src/")
            .append(componentName)
            .append(".vhdl")
            .append(System.getProperty("line.separator"));
```

I don't know if there's an other way, but I could find a way to not use a fixed String

Reply:
I second @BFH-ktt1 - hardcoded strings are asking for trobules. But I think that we can meet in the middle here because

    "vcom -reportprogress 300 -work work ../src/LogisimVhdlSimComp_0.vhdl"

is built by code:

```
compFiles.append("vcom -reportprogress 300 -work work ../src/") .append(componentName)
            .append(".vhdl")
            ...
```

you should build the string you match against:

    else if (line.contains("\"vcom -reportprogress 300 -work work ../src/LogisimVhdlSimComp"))

the same way which would prevent problems when if any case we decide `componentName` to be differently built.
```

Reply:
@esseivan: This comment by @MarcinOrlowski still needs to be addressed before merging -- unless he is fine with your proposal in the meantime.